ios/Podfile.lock

```diff
@@ -272,11 +272,11 @@ PODS:
     - React-Core (= 0.63.2)
     - React-cxxreact (= 0.63.2)
     - React-jsi (= 0.63.2)
-  - Realm (5.4.7):
```

Comment:
could we keep these dependencies as they were before?

Reply:
yup! for context dev's and my podfiles.lock are off, i think theres been a recent update that didnt have the dependencies resolved consistently.